modules/survey_accounts/test/survey_accountsTest.php

```diff
@@ -15,16 +16,123 @@
 class survey_accountsTestIntegrationTest extends LorisIntegrationTest
 {
     /**
+     * Insert testing data
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+         $window = new WebDriverWindow($this->webDriver);
+         $size = new WebDriverDimension(1024,768);
+         $window->setSize($size);
+         $this->DB->insert(
+            "psc",
+            array(
+             'CenterID' => '55',
+             'Name' => 'TESTinPSC',
+             'Alias' => 'test',
+             'MRI_alias' => 'test'
+            )
+        );
+          $this->DB->insert(
+            "candidate",
+            array(
+             'CandID'        => '999888',
+             'CenterID'      => '55',
+             'UserID'        => '1',
+             'PSCID'         => '8888',
+             'ProjectID'     => '7777'
+            )
+        );
+        $this->DB->insert(
+            "session",
+            array(
+             'ID'            => '111111',
+             'CandID'        => '999888',
+             'CenterID'      => '55',
+             'UserID'        => '1',
+             'MRIQCStatus'   => 'Pass',
+             'SubprojectID'  => '6666',
+             'Visit'         => 'In Progress'
+            )
+        );
+          $this->DB->insert(
+            "candidate",
+            array(
+             'CandID'        => '999999',
+             'CenterID'      => '55',
+             'UserID'        => '1',
+             'PSCID'         => '8889',
+             'ProjectID'     => '7777'
+            )
+        );
+        $this->DB->insert(
+            "session",
+            array(
+             'ID'            => '111112',
+             'CandID'        => '999999',
+             'CenterID'      => '55',
+             'UserID'        => '1',
+             'MRIQCStatus'   => 'Pass',
+             'SubprojectID'  => '6666',
+             'Visit'         => 'In Progress'
+            )
+        );
+        $this->DB->insert(
+            "participant_accounts",
+            array(
+             'SessionID'     => '111111',
+             'Email'         => 'TestTestTest@gmail.com',
```

Comment:
I think this should be an @example.com email instead of @gmail.com. The gmail version is probably someone's real email address.